client/wildcard/src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,63 +1,62 @@
+    placement?: TooltipPrimitive.TooltipContentProps['side']
 }
 
-const TOOLTIP_MODIFIERS: Popper.Modifiers = {
-    flip: {
-        enabled: false,
-    },
-    preventOverflow: {
-        boundariesElement: 'window',
-    },
-}
+/** Arrow width in pixels */
+const TOOLTIP_ARROW_WIDTH = 14
+/** Arrow height in pixel */
+const TOOLTIP_ARROW_HEIGHT = 6
 
-/**
- * Renders a Tooltip that can be positioned relative to a target element.
- *
- * This component should typically only need to be rendered once in a React tree.
- * If you need to attach a tooltip to an specific element, simply add the `data-tooltip` attribute to that element.
- */
-export const Tooltip: React.FunctionComponent<React.PropsWithChildren<TooltipProps>> = ({ className }) => {
-    const { subject, content, subjectSeq, placement = 'auto', delay } = useTooltipState()
-
-    const tooltipStyle = useMemo(() => getTooltipStyle(placement), [placement])
-
-    if (!subject || !content) {
-        return null
-    }
-
-    return (
-        <BootstrapTooltip
-            // Set key prop to work around a bug where quickly mousing between 2 elements with tooltips
-            // displays the 2nd element's tooltip as still pointing to the first.
-            key={subjectSeq}
-            isOpen={true}
-            target={subject}
-            placement={placement}
-            // in order to add our own placement classes we need to set the popperClassNames
-            // here is where bootstrap injects it's placement classes such as 'bs-tooltip-auto' automatically.
-            popperClassName={classNames(styles.tooltip, styles.show, className, tooltipStyle)}
-            arrowClassName={styles.arrow}
-            innerClassName={styles.tooltipInner}
-            // This is a workaround to an issue with tooltips in reactstrap that causes the entire page to freeze.
-            // Remove when https://github.com/reactstrap/reactstrap/issues/1482 is fixed.
-            modifiers={TOOLTIP_MODIFIERS}
-            delay={delay}
-        >
-            {content}
-        </BootstrapTooltip>
-    )
+// Handling the onPointerDownOutside event and preventing the default behavior allows us to keep the Tooltip content open
+// even if the trigger <span> was clicked; this allows buttons to be clicked and text to be selected without dismissing content.
+// Reference: https://github.com/radix-ui/primitives/issues/1077
+function onPointerDownOutside(event: Event): void {
+    event.preventDefault()
 }
+
+export const Tooltip: React.FunctionComponent<TooltipProps> = ({
+    children,
+    content,
+    defaultOpen = false,
+    placement = 'right',
+}) => (
+    <TooltipPrimitive.Root delayDuration={0} defaultOpen={defaultOpen}>
+        <TooltipPrimitive.Trigger asChild={true}>
+            <span role="presentation" className={styles.tooltip} onClick={event => event.preventDefault()}>
```

Comment:
```suggestion
            <span role="presentation" className={styles.tooltip}>
```

Do you need this? I think there would be no default behavior on the `span` that we'd need to prevent here 🤔 Did you run into any issues?

Reply:
This was also required to support keeping the Tooltip open properly when the trigger is clicked - it didn't seem to work correctly without it, though I did try that first (it is used in the example from Radix, though). I'll add an additional comment for this line to make that clear!